cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -120,7 +120,9 @@ function generate_prune_whitelist_flags() {
 # besides the default ones.
 extra_prune_whitelist=
 if [ -n "${KUBECTL_EXTRA_PRUNE_WHITELIST:-}" ]; then
-  extra_prune_whitelist=( "${KUBECTL_EXTRA_PRUNE_WHITELIST:-}" )
+  # Note: we cannot have ( "${KUBECTL_EXTRA_PRUNE_WHITELIST:-}" )
+  # as quotes break parsing space separated values
+  extra_prune_whitelist=( ${KUBECTL_EXTRA_PRUNE_WHITELIST:-} )
```

Comment:
Space splitting is an anti-pattern though :/

Reply:
We need a `# shellcheck disable=SC2206` here

Reply:
Replaced with `read -ra`